src/epoll.rs

```diff
@@ -65,11 +65,11 @@ impl Poller {
             PollMode::Oneshot,
         )?;
 
-        log::trace!(
-            "new: epoll_fd={}, event_fd={}, timer_fd={:?}",
-            poller.epoll_fd.as_raw_fd(),
-            poller.event_fd.as_raw_fd(),
-            poller.timer_fd
+        tracing::trace!(
+            epoll_fd= ?poller.epoll_fd.as_raw_fd(),
+            event_fd= ?poller.event_fd.as_raw_fd(),
+            timer_fd= ?poller.timer_fd,
```

Comment:
uhh why does it still use `?` and not `%`?

Reply:
Since the raw FD is just a number there shouldn't be a difference, although I can adjust it if it's an issue.